db/migrate/20150910170527_add_data_to_credit_card.rb

```diff
@@ -1,5 +1,9 @@
 class AddDataToCreditCard < ActiveRecord::Migration
   def change
-    add_column :spree_credit_cards, :data, :json
+    if ActiveRecord::Base.connection.adapter_name != 'SQLite' && ActiveRecord::Base.connection.supports_json?
```

Comment:
@kamui @allisonlarson Please pay attention that ``` supports_json? ``` has been merged here:
https://github.com/rails/rails/pull/21110
but it's still not part of the latest minor of ```active_record (4.2.5) ```.
It's a better version but still an edge one :smiley: